cudapoa/src/cudapoa_generate_consensus.cuh

```diff
@@ -282,7 +282,7 @@ __device__ void generateConsensus(uint8_t* nodes,
     consensus[consensus_pos] = '\0';
 }
 
-template <bool cuda_banded_alignment = false, typename SizeT>
```

Comment:
I think we need to run this through PEF to verify this. I removed it because it was bloating the calls file with multiple permutations with all the different configurations.

Reply:
I think the cudapoa microbenchmarks would be sufficient for that. we don't need to run the end to end pipeline for it.